### PR TITLE
[#6372] add irods_schema_messaging to gitignore

### PR DESCRIPTION
**Pull request by SwooshyCueb**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ packaging/unattended_setup.json
 .ycm_extra_conf.py
 compile_commands.json
 CMakeLists.txt.user
+
+# old submodule
+/irods_schema_messaging
```
Comment (korydraughn) [inline .gitignore:16]: Can the use of a leading forward slash be misinterpreted by `git`?

Comment (trel) [reply .gitignore:16]: seems okay?

https://stackoverflow.com/a/38433961

Comment (SwooshyCueb) [reply .gitignore:16]: The leading forward slash matches the root of the tree, which is what we want.

**Review by alanking — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**